bin/local-env/docker-compose.yml

```diff
@@ -23,9 +25,14 @@ services:
     depends_on:
       - mysql
 
+  wordpress-debug-log:
+    image: alpine
+    volumes:
+      - wordpress_data:/var/www/html
+    command: tail -n 0 -F /var/www/html/wp-content/debug.log
```

Comment:
This live-tails the `debug.log` file but _only lines which were logged after the command started_, so we don't need to worry about emptying the log file every time. Like everything else in the volume, it will be deleted whenever `env:stop` is run.